### ISSUE TITLE
Translation: Python version missing, tests will fail due to error message

### ISSUE THREAD

**Issue by nomennescio** [Codewars/codewars.com#1134]
When publishing my Python translation, suddenly the tests failed with error message:

Error: python version python not supported

I noticed that the python version was not shown anymore

https://www.codewars.com/kumite/59b460ba5227dd69dd000477/edit

**Comment by nomennescio** [comment]
Oh wow! When selecting a different language, I noticed that now Python has completely disappeared from the languages drop-down box.

**Comment by anter69** [comment]
@kazk  This issue is still standing: trying to "edit" the python translation gives `invalid python version` errors, and the version selection is totally missing:

![edit translation](https://user-images.githubusercontent.com/12795864/39255361-a49be028-48ac-11e8-9481-e5e12aef56c4.png)

Clicking on the language selector shows a list of languages **without python**:

![edit translation 2](https://user-images.githubusercontent.com/12795864/39255756-90149b76-48ad-11e8-939f-751852b2b8d7.png)

However, clicking on "fork" you get the correct controls: language selector show **only python** and the version selector is also correct:

![fork translation](https://user-images.githubusercontent.com/12795864/39255846-bed0b422-48ad-11e8-8115-fcd77cca35b4.png)

**Comment by kazk** [comment]
Yeah, that error message appears because the language version is set to `"python"`. Unfortunately, I have no idea why Codewars does that.

Related: #1412

**Comment by javatlacati** [comment]
Try the following test:

* create a kumite
* publish
* fork the kumite
* save second kumite
* edit second kumite
* publish second kumite
* edit published kumited

At this point I think the version should be gone.

**Comment by anter69** [comment]
A more simple workaround for the issue:
* unpublish
* (do your edits)
* publish

**Comment by anter69** [comment]
@kazk I suggest to remove "python" from the title, because it seems to affect all languages with multiple versions: python and js was already reported, and I just encountered the same issue in ruby.
